point-of-sale-app/inventory/src/main/java/com/google/abmedge/inventory/InventoryController.java

```diff
@@ -50,13 +94,92 @@ public String readiness() {
     return new ResponseEntity<>("ok", HttpStatus.OK);
   }
 
-  @RequestMapping("/")
-  public String home() {
-    return "Hello Anthos BareMetal - Inventory Controller";
+  @GetMapping(value = "/items", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<String> items() {
+    List<Item> inventoryItems = activeItemsType.equals(ALL_ITEMS) ?
+        activeConnector.getAll() : activeConnector.getAllByType(activeItemsType);
+    String jsonString = GSON.toJson(inventoryItems, new TypeToken<List<Item>>() {
+    }.getType());
+    return new ResponseEntity<>(jsonString, HttpStatus.OK);
+  }
+
+  @PostMapping(value = "/switch/{type}")
```

Comment:
Could you add a comment here, for what `/switch` does? what does it mean to change the active inventory type?

Reply:
Added comment in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/2/commits/ac4ed2a65d0081ed1cbd476ca3969cb6cac4b344